README.md

```diff
@@ -16,14 +48,61 @@ Python tools for photovoltaic degradation analysis.
 import rdtools
 ```
 
-For usage examples, look at the notebooks in rdtools/docs.
+The most frequently used functions are:
+
+```Python
+normalized, insolation = normalize_with_pvwatts(energy, pvwatts_kws)
+  '''
+  Inputs: Pandas time series of raw energy, PVwatts dict for system analysis 
+    (poa_global, P_ref, T_cell, G_ref, T_ref, gamma_pdc)
+  Outputs: Pandas time series of normalized energy and POA insolation
+  '''
+```
+
+```Python
+poa_filter(poa); tcell_filter(Tcell); clip_filter(power); 
+csi_filter(insolation, clearsky_insolation)
+  '''
+  Inputs: Pandas time series of raw data to be filtered.
+  Output: Boolean mask where `True` indicates acceptable data
+  '''
+```
+
+```Python
+aggregation_insol(normalized, insolation, frequency='D')
+  '''
+  Inputs: Normalized energy and insolation
+  Output: Aggregated data, weighted by the insolation.
+  '''
+```
+
+```Python
+degradataion_year_on_year(aggregated)
+  '''
+  Inputs: Aggregated, normalized, filtered time series data
+  Outputs: Tuple: `yoy_rd`: Degradation rate 
+    `yoy_ci`: Confidence interval `yoy_info`: associated analysis data
+  '''
+```
+
+For additional usage examples, look at the notebooks in [rdtools/docs](./docs/degradation_example.ipynb).
```

Comment:
I suggest we point out the example notebook in a more prominent spot. Perhaps both right up in the "About" section and then again at the _beginning_ of the "Usage" section.

Reply:
OK